### ISSUE TITLE
Coinswitch Issues

### ISSUE THREAD

**Issue by astupidmoose** [btcpayserver/btcpayserver#503]
I've found a couple issues with Coinswitch (cc @Kukks):

1. When pay with coinswitch is clicked, the changelly button disappears. Screen needs refreshing before it comes back even if coinswitch is closed. 

2. (Browser Related) - If selected as a popup, many browsers will block this. Not really something we can control, but perhaps something that needs documentation

3. Coinswitch itself seems to have an issue calculating its own rate. Example: Coinswitch quotes "171.29 UBQ", then proceeds to ask for "169.309 UBQ". It quotes and then immediately asks for less. Its very strange, but not BTCPay related. 

4. Coinswitch BTCPay merchant setup page is broken, their setup process has errors which prevent merchants from properly signing up right now, again their issue not ours.

**Comment by ankitarora121** [comment]
Hi,

Regarding #3 - thanks for the heads up. We missed that bug :). Fixed. You might still see a minor difference due to market fluctuation. 
In the coming week, we will be adding fixed-price conversions to our widget which would guarantee fixed rates for 10 minutes. 

Regarding #4 - We noticed and fixed the issue right around the time you posted this. Thanks.

Feel free to contact me directly if anyone has specific CoinSwitch related queries.
